Source/SwiftyJSON/SwiftyJSON.swift

```diff
@@ -1236,6 +1236,8 @@ extension NSNumber {
     }
 }
 
+#if os(Linux)
+#else
```

Comment:
This gives an impression that Linux platform is not finished.

I would prefer more explicit unary `not` comparison here.

```
#if !os(Linux)
#endif
```